python/openai/README.md

```diff
@@ -315,3 +315,15 @@ available arguments and default values.
 
 For more information on the `tritonfrontend` python bindings, see the docs
 [here](https://github.com/triton-inference-server/server/blob/main/docs/customization_guide/tritonfrontend.md).
+
+## Model Parallelism Support
+
+- For multi-GPU models (ex: Tensor Parallelism > 1), here is the current support matrix:
```

Comment:
```suggestion
```

I think with the heading you can remove the sentence -

Reply:
done